data-raw/england-icb-secondary-care.R

```diff
@@ -91,7 +92,8 @@ criteria_to_reside_icb <-
   ) |>
   left_join(available_icb_beds) |>
   mutate(perc_not_meet_criteria = do_not_meet_criteria_to_reside / available_beds) |>
-  filter(date >= max(date) %m-% months(3)) |>
+  # filter(date >= max(date) %m-% months(3)) |>
+  filter(date >= as.Date("2023-01-01") & date <= as.Date("2023-03-31")) |>
```

Comment:
Why did you replace the date filter with hard-coded limits? This could be a source of mistakes if these limits are not updated when updating data sets.